src/ngx-smart-modal/src/ngx-smart-modal.scss

```diff
@@ -60,7 +62,7 @@ body.dialog-open {
 
 // Dialog modal
 .nsm-dialog {
-  position: relative;
+  position: absolute;
```

Comment:
Maybe only change the position to relative when the input draggable is set to true. You can do it easily on the markup in the ngx-smart-modal.componant.ts

Reply:
IMO, given that you're setting the `draggable` class at: `[class.draggable]="draggable && draggableEdges"`, I think you can add the `position: absolute;` CSS rule at the line 172 (inside your `.draggable` selector). 😉👍

Reply:
Yes the solution of @maximelafarie is quite better. More lisibility 😄

Reply:
I'm positive that you meant absolute, when draggable is set to true.
It used to be relative, but I changed it to absolute after implementing draggable feature.

Reply:
> IMO, given that you're setting the `draggable` class at: `[class.draggable]="draggable && draggableEdges"`, I think you can add the `position: absolute;` CSS rule at the line 172 (inside your `.draggable` selector). 😉👍

I have fixed it, but not like this. I'm going to fix it now.

Reply:
> IMO, given that you're setting the `draggable` class at: `[class.draggable]="draggable && draggableEdges"`, I think you can add the `position: absolute;` CSS rule at the line 172 (inside your `.draggable` selector). 😉👍

Actually no. I'm adding .draggable on #nsmContent, not #nsmDialog. Position that we are talking about is on #nsmDialog element.